pkgs/by-name/fi/firefly-iii/package.nix

```diff
@@ -33,14 +36,23 @@ let
   };
 in
 
-phpPackage.buildComposerProject (finalAttrs: {
```

Comment:
Given that the docs use `finalAttrs` let's keep it that way. I know that when I'm trying to copy nix expressions myself being able to match `finalAttrs` with what's given in the docs is easier than understanding what `_` does.

Reply:
Good point. I've added it again.